src/main/java/org/broadinstitute/hellbender/tools/exome/Pulldown.java

```diff
@@ -34,7 +35,8 @@ public Pulldown(final File inputFile, final SAMFileHeader header) {
      * AllelicCounts.  This IntervalList is modifiable and does not change with the state of the Pulldown.   */
     public IntervalList getIntervals() {
         final IntervalList intervals = new IntervalList(header);
-        intervals.addall(getCounts().stream().map(AllelicCount::getInterval).collect(Collectors.toList()));
+        intervals.addall(getCounts().stream().map(AllelicCount::getInterval)
```

Comment:
`AllelicCount::getInterval` already makes a defensive copy, so I think the second `map` is redundant.